_posts/2021-05-16-overview-of-architecture-review-processes.md

```diff
@@ -0,0 +1,253 @@
+---
```

Comment:
General feedback:

1. The first glance is a wall of text. One idea to brighten it up would be to create/add a few illustrations that underline or support your text. Two or three might be full enough.
2. What do you think about a 3 sentence "tl;dr" at the beginning of the article?
3. The article has multiple long sentences. Small/short sentences are easier to read and understand. Maybe you want to consider ease out a few sentences.
4. The reading flow is getting easier. The first part is hard to read. The middle section is getting easier. And the design doc part is a good flow. Not sure what the root cause is. One observations: In the design doc part, the quotes are embedded into the reading flow/in the story.
5. I have the feeling you are using a lot of filler words. Some words/parts can be removed without removing the meaning/context. This would make the text crisper.
6. I love that you provide decent references. Thanks for doing so. Very rare.

For point 3, 4 and 5, tools like https://hemingwayapp.com/ and Grammarly can help.

Overall a pretty solid article.
A little bit hard/long to read, but while tackling 4, 5, and 6, it is getting shorter.

What I would love to see:
1. A direct comparision for those three (maybe in a table?)
2. Your personal experience and opinion when to use what
3. One example aerchitecture and applied every process on it

I am pretty sure those requests should not go into this article.
They might be food for thought for future articles.

Good work, Steffen!

Reply:
Added 3 figures about the processes. If you could take a look, would be very cool ❤️

Reply:
I also added a basic table to summarize the three different process in the end.